mibs/huawei/HUAWEI-POE-MIB.mib

```diff
@@ -0,0 +1,720 @@
+
```

Comment:
Can you rename this to drop the `.mib` please.

Reply:
OK